### ISSUE TITLE
[Feature Request]: Option to automatically switch to Full Window mode when video is playing

### ISSUE THREAD

**Issue by kodiakcreativity** [FreeTubeApp/FreeTube#2756]
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for open and closed issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.

### Problem Description

I like to watch videos in Full Window mode ( not full screen, but full window only) but to do so I must click the full window button each time as the video begins to play.

### Proposed Solution

I would like to see an option to automatically use full window mode. My preferred scenario would be this: with this option enabled, when I click on a video to play that video, as soon as the video begins playing the display would switch to full window mode. It should stay in that mode if I pause or otherwise interact with the video, normal mode can be reestablished at any time by clicking the full window button manually when already in full window mode.
additionally for usability, I'd like the video mode to switch back to normal automatically when the video is finished, because once the video is done you're likely to want the UI controls back to pick another one.

### Alternatives Considered

Instead of an option, it could remember this preference. if I put the video into full window mode, it could automatically play all videos in full window mode unless I click the full window mode button again to restore to normal mode, so no setting would be needed. I would still prefer the UI to go back to normal mode to show all controls once the video has stopped playing.
another alternative which could be useful is a way to request full window mode only when the video is actually playing, it could go back to normal when paused just in case it was paused because the user wants to stop the video and pick a different one.

### Issue Labels

ease of use improvement, new feature

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of #2295
